src/types.d.ts

```diff
@@ -18,6 +18,7 @@ export type Rule = {
 	// be OR'd together in this property.
 	defaultConfig?: ESRuleConfigValue;
 	url?: string;
+	message?: string;
```

Comment:
In RuleDescribeAction, we referenced the 'message' field dynamically... but now i'm referencing it statically. This wasn't possible because we were missing the message field from our type here. So I fixed this.

Reply:
Can you help me understand what a "message" would be in a Rule object? "description" would be the description of the rule and the Violation type should have information that the engine returns, so I'm not sure what a "message" would be here.

Reply:
For example (even before my changes):
> sf scanner rule describe -n ApexCRUDViolation
 Would give a bunch of information including:
 message:        Validate CRUD permission before SOQL/DML operation or enforce user mode